src/axis.js

```diff
@@ -44,6 +59,11 @@ export default (d3, config, xScale) => {
             .axisTop(xScale)
             .tickFormat(d => tickFormat(d, formats, d3));
 
+        const breakpoint = getBreakpointLabel(window.innerWidth) || 'extra';
```

Comment:
Can't we do it in the `resize` event instead? We don't need to recompute it everytime if window size is kept the same.